### ISSUE TITLE
How is Shotcut different from Kdenlive?

### ISSUE THREAD

**Issue by unfa** [mltframework/shotcut#465]
I apologize if this is a bad place to ask such a question but I am thinking about this for weeks and I have to ask it finally.

**What differs Shotcut from Kdenlive?**

Both programs use MLT as their video processing backend, both are non-linear video editors, (I guess) both use the Qt framework for GUI and both are open-source and free software. Both even look and feel similar to me in many ways.

I'm not asking about the details. I'm asking for the reason to start a new project, instead of helping develop an existing one. I guess there has to be a major difference in the approach taken by the Shotcut team - otherwise I guess the devs would rather just join development of Kdenlive, right?

So what is it? What makes Shotcut and Kdenlive fundamentally different?
I hope that'll let me better understand which one should I use for my purposes.

(I could also ask how is Shotcut different from Openshot, but Openshot no longer uses MLT)

**Comment by ddennedy** [comment]
Shotcut was born to be cross-platform and easily to download and run. My experience with Kdenlive on macOS and MacPorts and what I had seen of other KDE ports to Windows made me specifically not want to use KDE. I wish Kdenlive had chosen to remain focused on Linux and BSD. Shotcut also came at a time when I needed to tease out the new cross-platform capabilities of MLT in conjunction with new technologies in the form of WebVfx, Movit, and Qt5 while Kdenlive development appeared to have died. I did not wish to exercise those things in a large complex codebase I do not know that well.
